app/views/customer_mailer/customer_email.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class = "container ">
+  <div class = "whith-100">
+    <h2 class = "text-center mt-5"><%= t "mail.confirm" %></h2>
+    <p class = "mt-3"><%= t "mail.guide_all_1" %></p>
+    <h4 class = "mt-5"><%= t "mail.code_seat" %></h4>
+    <% @tickets.each do |ticket| %>
+      <span class = "bg-warning"><%= ticket.code %></span>
+    <% end %>
+    <p class = "mt-5"></p>
+    <% @tickets.each do |ticket| %>
```

Comment:
đoạn này lặp lại hơi bị nhiều lần cái mảng tickets đó nhoa

Reply:
dạ vì em muốn in ra mã bên trên còn vé in xuống dưới ạ

Reply:
c nghĩ là nên pluck code của ticket ra vì những thông tin đó như nhau mà đúng hơm nhể?